Validation_Tests/ValidationsTests/IsNotEmptyValidationTests/IsNotEmptyValidation_RunnerTests.GivenAValidator_WithErrors_ThenEveryFieldHasErrors.approved.json

```diff
@@ -0,0 +1,160 @@
+﻿{

```

Comment:
This is no longer the correct format, for this file.

You should consider doing a rebase on master.
And re-running all the tests.

Reply:
Why a rebase?